### ISSUE TITLE
cli: pipcook run not working with url

### ISSUE THREAD

**Issue by yorkie** [alibaba/pipcook#744]
```sh
pipcook run https://raw.githubusercontent.com/alibaba/pipcook/main/example/pipelines/bayes.v2.json
✖ read pipeline file error: ENOENT: no such file or directory, open 'https://raw.githubusercontent.com/alibaba/pipcook/main/example/pipelines/bayes.v2.json'
```

It seems reading the URL from the file system :(

**Comment by yorkie** [comment]
Installed via `npm install @pipcook/cli@rc -g`, but still got failed, and the `pipcook --version` shows `2.0.0-beta.1`.

**Comment by yorkie** [comment]
The following values are inconsistent:

- /usr/local/lib/node_modules/@pipcook/cli/dist/package.json#version `2.0.0-beta.1`
- /usr/local/lib/node_modules/@pipcook/cli/package.json#version `2.0.0-rc.1`

**Comment by FeelyChau** [comment]
File protocol is not supported at rc1, see: #742

**Comment by FeelyChau** [comment]
`2.0.0-rc-2` has been published, plz reinstall `@pipcook/cli@rc` and retry to run pipeline from url.

**Comment by FeelyChau** [comment]
> Installed via `npm install @pipcook/cli@rc -g`, but still got failed, and the `pipcook --version` shows `2.0.0-beta.1`.

This bug could be fixed by #749 , without this PR, we need to rebuild the CLI after we tag it by `learn`, it's so easy to miss the new version which updated by `learn`.

**Comment by yorkie** [comment]
Thank you, it works for me :)
